src/system/user.cr

```diff
@@ -5,8 +5,10 @@ require "crystal/system/user"
 # Users can be retrieved by either username or their user ID:
 #
 # ```
+# require "system/user"
+#
 # System::User.find_by name: "root"
-# System::User.find_by id: "1000"
+# System::User.find_by id: "0"
```

Comment:
I changed the value from 1000 to 0 for two reasons.

1. This explains "two ways" of getting user data by **id** and **name**. Therefore, the intention of the code becomes clearer when the target is the same data.

2. I am using docker to verify the sample code, and id = 1000 does not exist in the container, so spec fails in macro level.